src/app/components/footer/footer.component.html

```diff
@@ -0,0 +1,48 @@
+<footer class="footer">
+  <div class="footer__content">
+    <div class="footer__logo">
+      <img src="assets/logo.png" alt="Logo de Jean-Luc Laffuge" />
+      <h4 class="footer__title">Cinq de cœur</h4>
+    </div>
+
+    <div class="footer__contact">
+      <h4 class="footer__subtitle">Contact</h4>
+      <a class="footer__text" [href]="'https://www.google.com/maps/search/' + encodeAddress(contactDetails.address)" target="_blank" rel="noopener noreferrer">
+        {{ contactDetails.address }}
+      </a>
+      <a class="footer__text" [href]="'tel:' + contactDetails.phone">
+        {{ contactDetails.phone }}
```

Comment:
Maybe we need to remove it to avoid spam the can see it on the booking link

Reply:
I've encoded in Base64, let me know is it's ok for you !

Reply:
Added a Captcha : 

I couldn't find the keys you'd put in the constants, which told me that the localhost wasn't in the authorized domains. 
So I tried to join the project with the link you gave me earlier, but I couldn't find where it was. 
So, with my personal account I added the domain localhost and cinqdecoeur on another personal captcha and it works. 
So you'll have to put back the right keys for the production and your analytics.

Reply:
Ok so o, localhost I have this error : 
![image](https://github.com/user-attachments/assets/700651a6-e158-49b3-8922-6555b3247d53)

Reply:
it works with `http://localhost:4200/` but not with `http://127.0.0.1:4200/`

Reply:
Ok so I merge it but we need to see together how you created the captcha key !